include/boost/numeric/odeint/stepper/bulirsch_stoer_dense_out.hpp

```diff
@@ -391,7 +391,7 @@ class bulirsch_stoer_dense_out {
     }
 
 
-private:
+protected:
```

Comment:
Is it necessary to mark everything as protected instead of private, or would some subset of members be sufficient?

Reply:
You are right. Marking m_max_dt as protected is sufficient.